bigbluebutton-html5/imports/api/whiteboard-multi-user/server/handlers/modifyWhiteboardAccess.js

```diff
@@ -1,37 +0,0 @@
-import { check } from 'meteor/check';
```

Comment:
I am confused now :confused: 
Do you think we don't need this file anymore?
May I ask why?